index.js

```diff
@@ -0,0 +1,123 @@
+module.exports = {
+    "plugins": [
+        "import"
+    ],
+    "extends": "eslint:recommended",
+    "rules": {
+        "no-unsafe-negation": "error",
+        "curly": [
+            "error",
+            "all"
+        ],
+        "default-case": "error",
+        "dot-location": [
+            "error",
+            "property"
+        ],
+        "dot-notation": "error",
+        "eqeqeq": "error",
+        "no-caller": "error",
+        "no-else-return": "error",
+        "no-empty-function": "error",
+        "no-eq-null": "error",
+        "no-extra-bind": "error",
+        "no-floating-decimal": "error",
+        "no-global-assign": "error",
+        "no-implicit-coercion": "error",
+        "no-implicit-globals": "error",
+        "no-implied-eval": "error",
+        "no-iterator": "error",
+        "no-lone-blocks": "error",
+        "no-multi-spaces": "error",
+        "no-new-func": "error",
+        "no-new-wrappers": "error",
+        "no-octal-escape": "error",
+        "no-param-reassign": [
+            "error",
+            {
+                "props": false
+            }
+        ],
+        "no-proto": "error",
+        "no-return-assign": [
+            "error",
+            "always"
+        ],
+        "no-self-assign": [
+            "error",
+            {
+                "props": false
+            }
+        ],
+        "no-self-compare": "error",
+        "no-sequences": "error",
+        "no-throw-literal": "error",
+        "no-useless-call": "error",
+        "no-useless-concat": "error",
+        "no-useless-escape": "error",
+        "no-with": "error",
+        "radix": [
+            "error",
+            "as-needed"
+        ],
+        "strict": [
+            "error",
+            "safe"
+        ],
+        "no-shadow-restricted-names": "error",
+        "no-use-before-define": "error",
+        "array-bracket-spacing": [
+            "error",
+            "never"
+        ],
+        "block-spacing": "error",
+        "brace-style": "error",
+        "comma-dangle": [
+            "error",
+            "never"
+        ],
```

Comment:
[Trailing function parameter commas](https://github.com/tc39/proposal-trailing-function-commas) are [coming next year](https://github.com/tc39/proposals/blob/master/finished-proposals.md), so this would be great at that time, definitely.

Until then, I'm happy to enforce trailing commas, I don't mind much. I only see it a problem in Angular inline-array dependency invocation, which would be solved by trailing function commas:
```js
module.factory('thing', [
    'dep1',
    'dep2',
    function (
        dep1,
        dep2
    ) {
        dep1.doAThingWith(dep2);
    },
]);
```
where the only place one can't put a comma is after the second `dep2`.